lib/import/brca/providers/royal_marsden/royal_marsden_handler.rb

```diff
@@ -98,15 +103,29 @@ def process_teststatus(genotype, record)
               @logger.debug 'UNABLE TO DETERMINE TESTSTATUS'
               return
             end
-
+            nonpathvarclass = nil
             teststatus = record.raw_fields['teststatus']
+            if record.raw_fields['variantpathclass'].nil?
```

Comment:
I think you can use safe navigation here

replace

```
if record.raw_fields['variantpathclass'].nil?
Comment
I think you can use safe navigation here

 if record.raw_fields['variantpathclass'].nil?
  nonpathvarclass = nil
  else
  nonpathvarclass = record.raw_fields['variantpathclass'].downcase.strip
end
```
nonpathvarclass = record.raw_fields['variantpathclass']&.downcase&.strip

Reply:
Changes made in commit [1c81ce7](https://github.com/NHSDigital/data_management_system/pull/67/commits/1c81ce76eb14751540b1c89c985963e3bc033f91). I checked that the variant counts still match.